### PR TITLE
Fix database schema, add logging and helper functions to make modules work

### PR DESCRIPTION
**Pull request by austenmc**
The juicy bits are setting cascade=remove on properties and changing the association type for CharacterViewpoint::scene.

### CODE REVIEW

File: src/Tools/Model/PropertyManager.php [hunk 1]
```diff
@@ -32,6 +32,17 @@ public function getProperty(string $name, $default = null)
         }
     }
 
+    public function unsetProperty(string $name)
```
Comment (Vassyli) [inline src/Tools/Model/PropertyManager.php:35]: We may could think about the naming scheme for functions like this. removeProperty sounds like more work than just "unset" and may hint that it's connected with an actual database query.

File: src/ModuleManager.php [hunk 2]
```diff
@@ -74,6 +73,7 @@ public function register(LibraryConfiguration $library)
 
             // Run the module's onRegister handler.
             $class::onRegister($this->g, $m);
+            $m->save($this->g->getEntityManager());
```
Comment (Vassyli) [inline src/ModuleManager.php:81]: Does this solve the issue of a half-installed module?

**Review by Vassyli — COMMENTED** [review]
It isn't that big. And it's fine. Ship it.

File: src/ModuleManager.php (hunk 2 above)
Comment (austenmc) [reply src/ModuleManager.php:81]: No. It's just that after you register a module, you expect it to be persisted to the DB.

**Pull request merged**